### ISSUE TITLE
Few conflicts with other libraries in alpha6

### ISSUE THREAD

**Issue by samirsa** [metasoarous/oz#114]
First of all, thanks for OZ. It is a great way to bring in the power of Clojure with Vega.

I am facing some issues with some library (which I am using) conflicts based on what version of Oz I am using, and have tried a few combinations. But, am not able to pinpoint the error. So, I will list them all and hope it helps someone who is more well-versed with library conflicting issues (as you can guess, i certainly am not)

- alpha6 (the latest available) used to give a taoensso.encore related error, and so I had moved back to alpha5 (did not spend a lot of time looking into it). 

- Was able to get alpha5 to work with  [org.clojure/core.cache "0.8.2"] (a library that I was using for other stuff). 

- alpha6 seems to work by adding [org.clojure/tools.reader "1.3.2"] and [respeced "0.0.1"], but I need to remove my core.cache dependency, otherwise I an error
   No implementation of method: :has? of protocol:
#'clojure.core.cache/CacheProtocol found for class: clojure.core.memoize.PluggableMemoization

- latest from github, gives the same error as well, if I include core.cache. 

- latest from github, and after removing core.cache, is stuck on the 'Waiting for first spec to load' screen. 

Have tried adding the core.async dependency in my project.clj as well, but still get the same error.

So, in summary there seem to be two issues. Some conflicts with core.cache for alpha6 onwards. and some issue in loading the spec (or showing anything at all) for the latest.

Would appreciate some help or tips in how i could try to proceed. If you could point me to possible error or solutions, I can try to fix. but I am no expert in resolving inter-library dependency issues. 

thanks and I hope we can solve this, so I can use the latest and greatest functionality instead of being stuck with alpha5 or alpha6.

**Comment by metasoarous** [comment]
Hi @samirsa. Thanks so much for posting this issue, and my deep apologies for letting it slip through the cracks.

We're currently now on `alpha29`. Please give that a shot and let me know if that resolves your issues. I've been meaning to get around to updating a whole host of dependencies generally speaking, so will report back when I get around to that.

Thanks again!

**Comment by metasoarous** [comment]
Hi @samirsa.

I haven't heard back from you in a while, so I'm going to go ahead and close this now. Please feel free to reopen if you find that you're still having the same problems. I think at the time I may have still been doing uberjars (see 9780a71, which comes after https://clojars.org/metasoarous/oz/versions/1.6.0-alpha6). The switch to regular jars has cleared up a lot of dependency related problems for users.

I do still intend to do a general updating of various dependencies, but unless I hear back from you to the contrary, I'm going to assume this issue has now been resolved.

Thanks again!
